agent/custom/general_navigation_action.py

```
@@ -624,306 +354,425 @@ def _iou(a, b):
 
-            mfaalog.error(f"[导航] 滑到底未找到目标关卡: {target_quest} (共滑动 {swipe_count} 次)")
+            # 步骤4: 超轮未达目标 -> 失败(无兜底)
+            mfaalog.info("[导航] 方案B超最大轮数未达目标, 无法导航(无兜底)")
             return CustomAction.RunResult(success=False)
 
         except Exception as e:
             mfaalog.error(f"[导航] 严重错误: {str(e)}")
             return CustomAction.RunResult(success=False)
 
+    @staticmethod
+    def _iou2(a, b):
```

Ruff (0.16.1)

[warning] Missing return type annotation for staticmethod `_iou2`

(ANN205)